travis/install_pg

```diff
@@ -15,7 +15,7 @@ packages="postgresql-common libedit-dev libpam0g-dev libselinux1-dev"
 # we set PGVERSION to 10x of the Citus version when testing Citus, so
 # only install PostgreSQL proper if it's 10 or lower
 if [ "${PGVERSION//./}" -le "100" ]; then
-    packages="$packages postgresql-$PGVERSION postgresql-server-dev-$PGVERSION"
+    packages="$packages postgresql-$PGVERSION postgresql-server-dev-$PGVERSION postgresql-contrib-$PGVERSION "
```

Comment:
Is this feature will not bbe supported in > PG10

Reply:
moved to previous locations as you suggested